common/errors.h

```diff
@@ -213,6 +213,9 @@ enum retcode_t {
   RC_CONSENSUS_EXIT_PROBABILITIES_INVALID_ENTRYPOINT =
       0x06 | RC_MODULE_CONSENSUS_EXIT_PROBABILITIES | RC_SEVERITY_MAJOR,
 
+  RC_CONSENSUS_NULL_BUNDLE_PTR =
```

Comment:
Does it make sense to scope this kind of generic error to consensus? Can't we just have a `RC_NULL_PTR` error and use that here?

Reply:
yes we can